stake-pool/cli/src/main.rs

```diff
@@ -375,6 +654,63 @@ fn command_list(config: &Config, pool: &Pubkey) -> CommandResult {
     Ok(None)
 }
 
+fn command_update(config: &Config, pool: &Pubkey) -> CommandResult {
+    // Get stake pool state
+    let pool_data = config.rpc_client.get_account_data(&pool)?;
+    let pool_data: StakePool = PoolState::deserialize(pool_data.as_slice())
+        .unwrap()
+        .stake_pool()
+        .unwrap();
+    let validator_stake_list_data = config
+        .rpc_client
+        .get_account_data(&pool_data.validator_stake_list)?;
+    let validator_stake_list_data =
+        ValidatorStakeList::deserialize(&validator_stake_list_data.as_slice())?;
+
+    let epoch_info = config.rpc_client.get_epoch_info()?;
+
+    let accounts_to_update: Vec<&Pubkey> = validator_stake_list_data
+        .validators
+        .iter()
+        .filter_map(|item| {
+            if item.last_update_epoch >= epoch_info.epoch {
+                None
+            } else {
+                Some(&item.validator_account)
+            }
+        })
+        .collect();
+
+    let mut instructions: Vec<Instruction> = vec![];
+
+    for chunk in accounts_to_update.chunks(MAX_ACCOUNTS_TO_UPDATE) {
+        instructions.push(update_list_balance(
```

Comment:
We should document this better on our side, but this might need to be broken up in multiple transactions if there are too many instructions.  You can see the maximum size of a compiled transaction at https://github.com/solana-labs/solana/blob/master/sdk/src/packet.rs#L13

Reply:
This seems like not much and we do need to split transaction into several. Will leave it to the next PR if you do not mind.

Reply:
No problem